CONFIG.md

```diff
@@ -0,0 +1,52 @@
+# Configuration
+
+##### 10/14/19
+We've begun the migration to a new configuration framework.  The 'old' method (the `sip-communicator.properties` file and program arguments) will be supported throughout the transition and the bridge will be fully backwards-compatible with the old config.  No changes are necessary until we pull the plug on the old config completely, and we'll give ample warning before that time.
```

Comment:
I'd be even more generous about how long we'll continue to support the old config file format -- I'm not sure this text should even be threatening to remove it.

That said, I'd suggest adding something to the effect of "new configuration options will only be supported in the new config file."

Reply:
i do like mentioning new config props will only be in the new file, will add that.

as far as the warning about deprecation...why not set the stage now?  i don't want to coddle people into thinking it'll be around forever, and i give no specific time (just mentioning that it'll be after the transition and 'ample' warning will be given)

Reply:
Okay, I guess that's fine. I just don't want people to panic thinking that they need to do this tomorrow, but I guess the "ample warning" wording is enough for that.